### DISCUSSION TITLE
WWS Bad request

### DISCUSSION THREAD

**Discussion by Rodrigo-de-Mello** [sympa-community/sympa#1864] [Q&A] [answered]
Hi all,
Yesterday sympa stopped to work. we dont changed anything in server configurations/permissions.
when try to access the url i only receive 0400 bad request error.

services aparently are running:

sympa     334970       1  0 08:56 ?        00:00:00 /usr/bin/multiwatch -f 5 -- /usr/local/lib/sympa/cgi/wwsympa.fcgi
sympa     334971  334970  8 08:56 ?        00:00:01 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
sympa     334972  334970  8 08:56 ?        00:00:01 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
sympa     334973  334970  8 08:56 ?        00:00:01 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
sympa     334974  334970  8 08:56 ?        00:00:01 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
sympa     334975  334970  8 08:56 ?        00:00:01 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
sympa     335155       1  0 08:56 ?        00:00:00 /usr/bin/perl /usr/local/sbin/bulk.pl
sympa     335156       1  1 08:56 ?        00:00:00 /usr/bin/perl /usr/local/sbin/sympa_msg.pl
sympa     335157       1  1 08:56 ?        00:00:00 /usr/bin/perl /usr/local/sbin/bounced.pl
sympa     335158       1 26 08:56 ?        00:00:00 /usr/bin/perl /usr/local/sbin/task_manager.pl
sympa     335159       1  2 08:56 ?        00:00:00 /usr/bin/perl /usr/local/sbin/archived.pl

● wwsympa.service - Sympa Web Interface FastCGI backend
     Loaded: loaded (/etc/systemd/system/wwsympa.service; enabled; vendor preset: enabled)
     Active: active (running) since Thu 2024-07-11 08:56:37 -03; 19min ago
TriggeredBy: ● wwsympa.socket
   Main PID: 334970 (multiwatch)
      Tasks: 6 (limit: 9387)
     Memory: 470.6M
        CPU: 7.517s
     CGroup: /system.slice/wwsympa.service
             ├─334970 /usr/bin/multiwatch -f 5 -- /usr/local/lib/sympa/cgi/wwsympa.fcgi
             ├─334971 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
             ├─334972 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
             ├─334973 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
             ├─334974 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi
             └─334975 /usr/bin/perl /usr/local/lib/sympa/cgi/wwsympa.fcgi

Jul 11 08:56:37 pinatubo systemd[1]: Started Sympa Web Interface FastCGI backend.
Jul 11 08:56:39 pinatubo wwsympa[334973]: info main:: WWSympa started, process 334973
Jul 11 08:56:39 pinatubo wwsympa[334974]: info main:: WWSympa started, process 334974
Jul 11 08:56:39 pinatubo wwsympa[334972]: info main:: WWSympa started, process 334972
Jul 11 08:56:39 pinatubo wwsympa[334975]: info main:: WWSympa started, process 334975
Jul 11 08:56:39 pinatubo wwsympa[334971]: info main:: WWSympa started, process 334971

● sympa.service - SYMPA mailing list manager
     Loaded: loaded (/etc/systemd/system/sympa.service; enabled; vendor preset: enabled)
     Active: activating (start) since Thu 2024-07-11 09:16:15 -03; 26s ago
       Docs: man:sympa_msg(8)
    Process: 361142 ExecStartPre=/bin/mkdir -p /run/sympa (code=exited, status=0/SUCCESS)
    Process: 361148 ExecStartPre=/bin/chown sympa:sympa /run/sympa (code=exited, status=0/SUCCESS)
    Process: 361153 ExecStart=/usr/local/sbin/sympa_msg.pl (code=exited, status=0/SUCCESS)
      Tasks: 1 (limit: 9387)
     Memory: 73.5M
        CPU: 1.629s
     CGroup: /system.slice/sympa.service
             └─361348 /usr/bin/perl /usr/local/sbin/sympa_msg.pl

Jul 11 09:16:15 pinatubo systemd[1]: Starting SYMPA mailing list manager...
Jul 11 09:16:18 pinatubo sympa_msg[361153]: info main::_load() Configuration file read, default log level 0
Jul 11 09:16:18 pinatubo sympa_msg[361153]: notice Sympa::Process::daemonize() Starting sympa/msg daemon, PID 361348
Jul 11 09:16:18 pinatubo sympa_msg[361348]: notice main:: Sympa/msg 6.2.72 Started
Jul 11 09:16:18 pinatubo systemd[1]: sympa.service: Can't open PID file /run/sympa/sympa_msg.pid (yet?) after start: Operation not permitted

sympa.log
Jul 11 09:16:18 pinatubo sympa_msg[361153]: info main::_load() Configuration file read, default log level 0
Jul 11 09:16:18 pinatubo bulk[361151]: info main:: Configuration file read, default log level 0
Jul 11 09:16:18 pinatubo bulk[361151]: notice Sympa::Process::daemonize() Starting bulk daemon, PID 361346
Jul 11 09:16:18 pinatubo task_manager[361152]: info main:: Configuration file read, default log level 0
Jul 11 09:16:18 pinatubo task_manager[361152]: notice Sympa::Process::daemonize() Starting task_manager daemon, PID 361345
Jul 11 09:16:18 pinatubo task_manager[361152]: notice main:: Task_Manager 6.2.72 Started
Jul 11 09:16:18 pinatubo bulk[361346]: notice main:: Bulk 6.2.72 Started
Jul 11 09:16:18 pinatubo archived.pl: info main:: Configuration file read, default log level 0
Jul 11 09:16:18 pinatubo sympa_msg[361153]: notice Sympa::Process::daemonize() Starting sympa/msg daemon, PID 361348
Jul 11 09:16:18 pinatubo archived[361347]: notice main:: Archived 6.2.72 Started
Jul 11 09:16:18 pinatubo bounced.pl: info main:: Configuration file read, default log level 0
Jul 11 09:16:18 pinatubo bounced[361349]: notice main:: Bounced 6.2.72 Started
Jul 11 09:16:18 pinatubo sympa_msg[361348]: notice main:: Sympa/msg 6.2.72 Started

SYSLOG:
Jul 11 09:16:18 pinatubo systemd[1]: sympa-archived.service: Can't open PID file /run/sympa/archived.pid (yet?) after start: Operation not permitted
Jul 11 09:16:18 pinatubo systemd[1]: sympa-bulk.service: Can't open PID file /run/sympa/bulk.pid (yet?) after start: Operation not permitted
Jul 11 09:16:18 pinatubo systemd[1]: sympa-task_manager.service: Can't open PID file /run/sympa/task_manager.pid (yet?) after start: Operation not permitted
Jul 11 09:16:18 pinatubo systemd[1]: sympa-bounced.service: Can't open PID file /run/sympa/bounced.pid (yet?) after start: Operation not permitted
Jul 11 09:16:18 pinatubo systemd[1]: sympa.service: Can't open PID file /run/sympa/sympa_msg.pid (yet?) after start: Operation not permitted

/run/sympa permission:
drwxrwxrwx  2 sympa    sympa        60 Jul 11 08:56 sympa

/run/sympa content:
drwxrwxrwx  2 sympa    sympa      60 Jul 11 08:56 ./
drwxr-xr-x 48 root     root     1540 Jul 11 08:42 ../
srw-rw-rw-  1 www-data www-data    0 Jul 11 08:56 wwsympa.socket=

sympa.conf

lang    pt-br
domain  xxx.xxx.xxx
listmaster      listmaster@xxx.xxx.xxx.xxx
cookie  zzzzzzzzzzzzzzz
db_type mysql
db_name sympa
db_host localhost
db_user sympa
db_passwd       xxxxxxxxxxxxxx
db_port 3306
static_content_path     /usr/local/var/lib/sympa/static_content
css_path        /usr/local/var/lib/sympa/static_content/css
css_url /css-sympa
pictures_path   /usr/local/var/lib/sympa/static_content/pictures
pictures_url    /pictures-sympa
use_fast_cgi    1
wwsympa_url     https://xxx.xxx.xxx.xxx/wws
aliases_program postalias
sendmail /usr/sbin/sendmail

/var/log/apache2/error.log:
[Thu Jul 11 08:57:23.287365 2024] [proxy_fcgi:error] [pid 334409:tid 140250854585920] [client 172.21.74.20:21525] AH01059: error parsing URL //: Invalid host/port
[Thu Jul 11 09:16:24.517225 2024] [proxy_fcgi:error] [pid 334410:tid 140249866954304] [client 172.20.13.15:49777] AH01059: error parsing URL //: Invalid host/port

Any help are welcome.

**Comment by ikedas** [comment, marked as answer]
Duplicate of #1863 .

**Reply by Rodrigo-de-Mello** [reply]
Thanks... I found for AH01059 in QA and didnt found any information... 
Worked fine here.
